packages/orbit-components/src/InputSelect/InputSelect.ct-story.tsx

```diff
@@ -60,55 +60,73 @@ const advancedOptions = [
 export function InputSelectStory() {
   return (
     <div className="gap-md p-md flex flex-col">
-      <InputSelect options={simpleOptions} />
-      <InputSelect options={simpleOptions} label="Custom label" placeholder="Custom placeholder" />
+      <InputSelect options={simpleOptions} readOnly />
+      <InputSelect
+        options={simpleOptions}
+        label="Custom label"
+        placeholder="Custom placeholder"
+        readOnly
```

Comment:
I added `readOnly` prop to all of the stories and tests, because it has `false` value by default (even for smaller devices).